examples/slides/src/components/Thanks.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import Link from './Link';
+import './Thanks.css';
+import meme from '../assets/tomhanksmeme.jpg';
+
+export default class Thanks extends React.Component {
+  static MAX_HANKS = 40;
+
+  constructor() {
+    super();
+    this.state.message = 'T.hanks';
+    this.state.numOfHanks = 1;
```

Comment:
nit: prefer `this.state = {}` in constructor
nit: prefer instance variable syntax without constructor `state = { message: '', numOfHanks: 1 }`